src/tests/CombatCalc.test.ts

```diff
@@ -0,0 +1,15 @@
+import {expect, test} from "@jest/globals";
+import {generateEmptyPlayer} from "@/state";
+import {calculate, getMonster} from "@/tests/utils/TestUtils";
+
+test('Empty player against Abyssal demon', () => {
+    let player = generateEmptyPlayer();
+    let monster = getMonster("Abyssal demon", "Standard");
+    let result = calculate(player, monster);
+
+    expect(result.maxHit).toBe(11);
+    expect(result.dps).toBe(0.6668319969138353);
```

Comment:
Not sure on the level of precision we want in this. Currently it fails if it isn't exactly the same.

Reply:
[You can use `toBeCloseTo()` instead](https://jestjs.io/docs/expect#tobeclosetonumber-numdigits) so it doesn't have to be exactly identical. Given that [results have a fixed precision of 3 digits](https://github.com/weirdgloop/osrs-dps-calc/blob/f705dd355d406caa784eec24cf160c7dceb1de1c/src/app/components/results/ResultsContainer.tsx#L25), we'd probably be safe matching that. (if that approach is taken, let's export a var for this precision so it doesn't have to be hard-coded about)